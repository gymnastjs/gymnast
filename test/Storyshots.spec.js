import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  framework: 'react',
  configPath: './node_modules/picturebook/config',
})
