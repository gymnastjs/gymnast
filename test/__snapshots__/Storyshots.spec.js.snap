// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Grid Auto Flow 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <div
      className="grid col-12 grid--stretch"
    >
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          A
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          B
        </div>
      </div>
      <div
        className="col-4"
      >
        <div
          className="box3"
          style={Object {}}
        >
          C
        </div>
      </div>
      <div
        className="col-3"
      >
        <div
          className="box4"
          style={Object {}}
        >
          D
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box5"
          style={
            Object {
              "height": 100,
            }
          }
        >
          E
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box1"
          style={Object {}}
        >
          A
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          B
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box3"
          style={Object {}}
        >
          C
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box4"
          style={Object {}}
        >
          D
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box5"
          style={Object {}}
        >
          E
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Fraction 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <h1>
      Auto Size
    </h1>
    <div
      className="grid col-12"
    />
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 1
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 2
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 2
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 3
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 3
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 3
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 4
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 4
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 4
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          1 / 4
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 5
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 5
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 5
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          1 / 5
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box5"
          style={Object {}}
        >
          1 / 5
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box5"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 6
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box5"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 7
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box5"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box2"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box3"
          style={Object {}}
        >
          1 / 8
        </div>
      </div>
    </div>
    <h1>
      8 - auto
    </h1>
    <div
      className="grid col-12"
    >
      <div
        className="col-8"
      >
        <div
          className="box2"
          style={Object {}}
        >
          8
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          12 - 8 = 4
        </div>
      </div>
    </div>
    <h1>
      6 - 1/2 - 1/2
    </h1>
    <div
      className="grid col-12"
    >
      <div
        className="col-6"
      >
        <div
          className="box3"
          style={Object {}}
        >
          6
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box4"
          style={Object {}}
        >
          (12 - 6) / 2 = 3
        </div>
      </div>
      <div
        className="col"
      >
        <div
          className="box5"
          style={Object {}}
        >
          (12 - 6) / 2 = 3
        </div>
      </div>
    </div>
    <h1>
      Custom
    </h1>
    <div
      className="grid col-12"
    />
    <div
      className="grid col-12"
    >
      <div
        className="col-6"
      >
        <div
          className="box3"
          style={Object {}}
        >
          6 (fixed)
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Horizontal Align 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <div
      className="grid grid--left col-12"
    >
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          LEFT
        </div>
      </div>
    </div>
    <div
      className="grid grid--center col-12"
    >
      <div
        className="col-2"
      >
        <div
          className="box3"
          style={Object {}}
        >
          CENTER
        </div>
      </div>
    </div>
    <div
      className="grid grid--right col-12"
    >
      <div
        className="col-2"
      >
        <div
          className="box4"
          style={Object {}}
        >
          RIGHT
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-2"
      >
        <div
          className="box5"
          style={Object {}}
        >
          DEFAULT
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Nested 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <div
      className="grid col-12"
    >
      <div
        className="grid col-6"
      >
        <div
          className="box1 grid grid--no-bottom"
          style={Object {}}
        >
          <div
            className="col-6"
          >
            <div
              className="box3"
              style={Object {}}
            >
              C
            </div>
          </div>
          <div
            className="col-6"
          >
            <div
              className="box4"
              style={Object {}}
            >
              D
            </div>
          </div>
        </div>
      </div>
      <div
        className="grid col-6"
      >
        <div
          className="box2 grid grid--no-bottom"
          style={Object {}}
        >
          <div
            className="col-3"
          >
            <div
              className="box4"
              style={Object {}}
            >
              D
            </div>
          </div>
          <div
            className="col--offset-6 col-3"
          >
            <div
              className="box5"
              style={Object {}}
            >
              E
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="grid col-6"
      >
        <div
          className="box1 grid grid--no-bottom"
          style={Object {}}
        >
          <div
            className="col--offset-3 col-6"
          >
            <div
              className="box4"
              style={Object {}}
            >
              D
            </div>
          </div>
        </div>
      </div>
      <div
        className="grid col-6"
      >
        <div
          className="box2 grid grid--no-bottom"
          style={Object {}}
        >
          <div
            className="col-4"
          >
            <div
              className="box4"
              style={Object {}}
            >
              D
            </div>
          </div>
          <div
            className="col--offset-4 col-4"
          >
            <div
              className="box5"
              style={Object {}}
            >
              E
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="grid col-6"
      >
        <div
          className="box2 grid grid--no-bottom"
          style={Object {}}
        >
          <div
            className="col-1"
          >
            <div
              className="box1"
              style={Object {}}
            >
              1
            </div>
          </div>
          <div
            className="col-1"
          >
            <div
              className="box2"
              style={Object {}}
            >
              2
            </div>
          </div>
          <div
            className="col-1"
          >
            <div
              className="box3"
              style={Object {}}
            >
              3
            </div>
          </div>
          <div
            className="col-1"
          >
            <div
              className="box4"
              style={Object {}}
            >
              4
            </div>
          </div>
          <div
            className="col-1"
          >
            <div
              className="box5"
              style={Object {}}
            >
              5
            </div>
          </div>
          <div
            className="col-1"
          >
            <div
              className="box1"
              style={Object {}}
            >
              6
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Offset 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <div
      className="grid col-12"
    >
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          +0
        </div>
      </div>
      <div
        className="col--offset-7 col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          +7
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col--offset-1 col-1"
      >
        <div
          className="box2"
          style={Object {}}
        >
          +1
        </div>
      </div>
      <div
        className="col--offset-5 col-1"
      >
        <div
          className="box2"
          style={Object {}}
        >
          +5
        </div>
      </div>
      <div
        className="col--offset-1 col-1"
      >
        <div
          className="box2"
          style={Object {}}
        >
          +1
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col--offset-2 col-1"
      >
        <div
          className="box3"
          style={Object {}}
        >
          +2
        </div>
      </div>
      <div
        className="col--offset-3 col-1"
      >
        <div
          className="box3"
          style={Object {}}
        >
          +3
        </div>
      </div>
      <div
        className="col--offset-3 col-1"
      >
        <div
          className="box3"
          style={Object {}}
        >
          +3
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col--offset-3 col-1"
      >
        <div
          className="box4"
          style={Object {}}
        >
          +3
        </div>
      </div>
      <div
        className="col--offset-1 col-1"
      >
        <div
          className="box4"
          style={Object {}}
        >
          +1
        </div>
      </div>
      <div
        className="col--offset-5 col-1"
      >
        <div
          className="box4"
          style={Object {}}
        >
          +5
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col--offset-4 col-1"
      >
        <div
          className="box5"
          style={Object {}}
        >
          +4
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          +0
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Sizing 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <div
      className="grid col-12"
    >
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={Object {}}
        >
          1
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          2
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-3"
      >
        <div
          className="box3"
          style={Object {}}
        >
          3
        </div>
      </div>
      <div
        className="col-3"
      >
        <div
          className="box3"
          style={Object {}}
        >
          3
        </div>
      </div>
      <div
        className="col-3"
      >
        <div
          className="box3"
          style={Object {}}
        >
          3
        </div>
      </div>
      <div
        className="col-3"
      >
        <div
          className="box3"
          style={Object {}}
        >
          3
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-4"
      >
        <div
          className="box4"
          style={Object {}}
        >
          4
        </div>
      </div>
      <div
        className="col-4"
      >
        <div
          className="box4"
          style={Object {}}
        >
          4
        </div>
      </div>
      <div
        className="col-4"
      >
        <div
          className="box4"
          style={Object {}}
        >
          4
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-6"
      >
        <div
          className="box5"
          style={Object {}}
        >
          6
        </div>
      </div>
      <div
        className="col-6"
      >
        <div
          className="box5"
          style={Object {}}
        >
          6
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col"
      >
        <div
          className="box1"
          style={Object {}}
        >
          Auto
        </div>
      </div>
    </div>
    <div
      className="grid col-12"
    >
      <div
        className="col-12"
      >
        <div
          className="box1"
          style={Object {}}
        >
          12
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Grid Vertical Align 1`] = `
<div
  className="gl-layout-root"
>
  <div
    className="gl-layout-content"
  >
    <h1>
      Item Align
    </h1>
    <div
      className="grid col-12"
    >
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={
            Object {
              "height": 300,
            }
          }
        >
           
        </div>
      </div>
      <div
        className="col--top col-2"
      >
        <div
          className="box2"
          style={Object {}}
        >
          TOP
        </div>
      </div>
      <div
        className="col--middle col-2"
      >
        <div
          className="box3"
          style={Object {}}
        >
          MIDDLE
        </div>
      </div>
      <div
        className="col--bottom col-2"
      >
        <div
          className="box4"
          style={Object {}}
        >
          BOTTOM
        </div>
      </div>
      <div
        className="col--middle col-2"
      >
        <div
          className="box3"
          style={Object {}}
        >
          MIDDLE
        </div>
      </div>
      <div
        className="col-2"
      >
        <div
          className="box5"
          style={Object {}}
        >
          DEFAULT
        </div>
      </div>
      <div
        className="col-1"
      >
        <div
          className="box1"
          style={
            Object {
              "height": 300,
            }
          }
        >
           
        </div>
      </div>
    </div>
    <h1>
      Grid Align
    </h1>
    <div
      className="grid col-12"
    >
      <div
        className="grid grid--top col-4"
        style={
          Object {
            "height": 300,
          }
        }
      >
        <div
          className="col-12"
        >
          <div
            className="box3"
            style={Object {}}
          >
            TOP
          </div>
        </div>
      </div>
      <div
        className="grid grid--middle col-4"
        style={
          Object {
            "height": 300,
          }
        }
      >
        <div
          className="col-12"
        >
          <div
            className="box3"
            style={Object {}}
          >
            MIDDLE
          </div>
        </div>
      </div>
      <div
        className="grid grid--bottom col-4"
        style={
          Object {
            "height": 300,
          }
        }
      >
        <div
          className="col-12"
        >
          <div
            className="box3"
            style={Object {}}
          >
            BOTTOM
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
